cybereason/boxes/servers.py

```diff
@@ -32,13 +32,13 @@ class PersonalizationBox(CRServerBox):
     Can run on any machine that can:
     Access the download server
     Untar the personalizer script
-    Run python 2.7
+    Run python 2.7 with packages 'protobuf' and 'google'
```

Comment:
Please add a link to each of the packages mentioned

Reply:
Done